data/json/itemgroups/science_and_tech.json

```diff
@@ -1,487 +0,0 @@
-[
-  {
-    "type": "item_group",
```

Comment:
What's up with removing all those item groups?

Reply:
I committed it to the wrong PR, sorry about that

Reply:
I'm still trying to figure this all out, is there not a way to delete the commit without removing the directory? I'll have to update that JSON to the original if not.

Reply:
I think `-git reset a4a0e52` should bring you back to `Update stationary.json` and remove the other stuff that came after

Reply:
thanks I'll try that

Reply:
I'm not sure how to input commands like that.

Reply:
I'm just going to overwrite with the original json